pkg/didcomm/protocol/didexchange/persistence/connection.go

```diff
@@ -12,6 +12,8 @@ import (
 	"errors"
 	"fmt"
 
+	"github.com/hyperledger/aries-framework-go/pkg/didcomm/protocol/didexchange"
+
```

Comment:
extra line

Reply:
removed this import no longer required with recent change merged